org-formation/_scp.yaml

```diff
@@ -0,0 +1,97 @@
+  RestrictUnusedRegionsSCP:
```

Comment:
this is not in use yet, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/010-scps/_tasks.yaml#L60-83